From 21b8522f1df5543f353901ce358bc9d3337ceacb Mon Sep 17 00:00:00 2001
From: Pier Luigi Fiorini <pierluigi.fiorini@gmail.com>
Date: Mon, 8 Oct 2012 20:28:57 +0200
Subject: [PATCH][BUILD] Add autogen.sh script for automatic configure
 generation.

The autogen.sh script is missing and this break the build.

Signed-off-by: Pier Luigi Fiorini <pierluigi.fiorini@gmail.com>
---
 autogen.sh | 13 +++++++++++++
 1 file changed, 13 insertions(+)
 create mode 100755 autogen.sh

diff --git a/autogen.sh b/autogen.sh
new file mode 100755
index 0000000..f05f9d4
--- /dev/null
+++ b/autogen.sh
@@ -0,0 +1,13 @@
+#!/bin/sh
+
+test -n "$srcdir" || srcdir=`dirname "$0"`
+test -n "$srcdir" || srcdir=.
+
+olddir=`pwd`
+cd "$srcdir"
+
+autoreconf --force --install || exit $?
+
+cd "$olddir"
+test -n "$NOCONFIGURE" || "$srcdir/configure" "$@"
+
-- 
1.7.12.2

